### ISSUE TITLE
Memberships no longer expire

### ISSUE THREAD

**Issue by AAnkudovich** [strangerstudios/pmpro-multiple-memberships-per-user#4]
Since activating this addon membership dont seem to expire anymore. 

Anyone have a suggestion for fixes?

**Comment by itibet** [comment]
Hi,
I have the same problem, I looked at the code and managed to fix it by hooking in my theme's functions the following action:
`add_action(	"pmpro_membership_post_membership_expiry",	"fix_pmpro_membership_post_membership_expiry", 10, 2 );`

`	public function fix_pmpro_membership_post_membership_expiry( $user_id, $membership_id )`
`	{`
`		pmpro_changeMembershipLevel( false, $user_id, "expired", $membership_id );`
`	}`

It works but I'm not sure there can't be some collateral effects, a note from the author would be great.
However, I noticed this addon is no longer offically available, probably there are other open issues.
Hope to help!

**Comment by AAnkudovich** [comment]
@itibet thank you very much will try it today

**Comment by AAnkudovich** [comment]
Above code solves the issue. Thank you @itibet

**Comment by AAnkudovich** [comment]
The above code solved the issue on Vagrant (Local set up)  but after publishing to live server no changes happened.

I see an error in my logs like so (this error may be caused by back ticks)
   
     WordPress database error Unknown column 'mlg.group' in 'where clause' for query \n\t\t\t\t\tSELECT level \n\t\t\t\t\tFROM wp_pmpro_membership_levels_groups AS mlg \n\t\t\t\t\tINNER JOIN wp_pmpro_membership_levels AS ml ON `ml.id` = `mlg.level` AND `ml.allow_signups` LIKE '1'\n\t\t\t\t\tWHERE `mlg.group` = 1 \n\t\t\t\t\tAND `ml.id` IN (2,3,4,6,8,9,10 )\n\t\t\t\t\tORDER BY level ASC made by require('wp-blog-header.php'), wp, WP->main, do_action_ref_array, WP_Hook->do_action, WP_Hook->apply_filters, pmpro_wp, require_once('/plugins/paid-memberships-pro/preheaders/checkout.php'), do_action('pmpro_after_checkout'), WP_Hook->do_action, WP_Hook->apply_filters, pmprommpu_pmpro_after_checkout, pmprommpu_get_levels_and_groups_in_order, referer: http://www.taxworld.co.uk/membership-account/membership-checkout/?level=4

**Comment by itibet** [comment]
Hi, that's another issue, see: https://github.com/strangerstudios/pmpro-multiple-memberships-per-user/issues/2

**Comment by AAnkudovich** [comment]
It is but solution provided only after back ticks were removed not sure if its related but should be all sorted now. Thank you
